src/main/java/de/unistuttgart/overworldbackend/data/Keybinding.java

```diff
@@ -0,0 +1,38 @@
+package de.unistuttgart.overworldbackend.data;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import de.unistuttgart.overworldbackend.data.enums.Binding;
+import java.util.UUID;
+import javax.persistence.*;
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+
+@Entity
+@Table(uniqueConstraints = { @UniqueConstraint(columnNames = { "player_user_id", "binding" }) })
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class Keybinding {
+
+    @Id
+    @GeneratedValue(generator = "uuid")
+    UUID id;
+
+    @JsonBackReference(value = "achievement-player")
+    @ManyToOne
+    Player player;
+
+    @Enumerated
+    Binding binding;
+
+    String key;
```

Comment:
Maybe add a regular expression to validate? So that no stupid string can be in there?